frontend/src/views/projects/roles/TeacherProjectView.vue

```diff
@@ -36,6 +37,11 @@ defineProps<{
         <div class="col-12 md:col-4">
             <ProjectMeter :project="project" />
         </div>
+        <div class="col-12">
```

Comment:
Can you add md: col-4 here too. And than maybe change the name of the button when working with small screens too please.